### PR TITLE
adding number of animals tested page

### PR DESCRIPTION
**Pull request by capJuan**
adding number of animals tested page

### CODE REVIEW

**Review by govTechSatish — DISMISSED** [review]

**Review by JulieLev — COMMENTED** [review]
Thoughts we touched on yesterday...

File: app/views/animals-tested.njk [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{% extends './layouts/layout.njk' %}
+
+{% set pageQuestion = "How many animals did the vet test?" %}
+{% set paragraph1 = "Enter the number of animals the vet tested. You can find it on the review summary the vet has given you." %}
+{% set paragraph2 = "You cannon use test results from tests carried out before you agreement start data or after your claim. If you do, your claim will be rejected." %}
```
[the remark this answers is not in the document]
Comment (capJuan) [reply app/views/animals-tested.njk:5]: done

File: app/lib/error-messages.js [hunk 2]
```diff
@@ -3,6 +3,10 @@ module.exports = {
     enterEmail: 'Enter an email address',
     validEmail: 'Enter an email address in the correct format, like name@example.com'
   },
+  animalsTested: {
+    enterNumber: 'Enter the number of animals tested',
+    numberMax: 'The number of animals tested should no exceed 99999'
```
[the remark this answers is not in the document]
Comment (capJuan) [reply app/lib/error-messages.js:8]: typo done,  Archie agreed with the upper limit

File: test/acceptance/features/dateOfTesting.feature [hunk 3]
```diff
@@ -25,7 +25,7 @@ Feature: claim journey landing page
     Then click on the back button
     Then user continue to claim
 
-Scenario: To validate the error in review completed "Date of review must be a real date"
+Scenario: To validate the error in review completed "The date the review was completed must be in the past"
```
[the remark this answers is not in the document]
Comment (capJuan) [reply test/acceptance/features/dateOfTesting.feature:28]: done

File: test/acceptance/page-objects/claimJourney-page.js [hunk 4]
```diff
@@ -46,7 +46,7 @@ const DATE_ERROR='a[href="#when-was-the-review-completed"]'
 const DATE_BLANK_ERROR_EXPECTED='Enter the date the vet completed the review'
 const CLICK_RADIO_BUTTON_ERROR='a[href="#when-was-endemic-disease-or-condition-testing-carried-out"]' 
 const CLICK_RADIO_BUTTON_ERROR_EXPECTED='Select if testing was carried out when the vet visited the farm or on another date'
-const INVALID_DATE_ERROR='Date of review must be a real date'
+const INVALID_DATE_ERROR='The date the review was completed must be in the past'
```
[the remark this answers is not in the document]
Comment (capJuan) [reply test/acceptance/page-objects/claimJourney-page.js]: changed back

**Pull request merged**